### ISSUE TITLE
Crash on mixpanel.track. Assertion failed: Property values must be of valid type (MixpanelType) and valid value.

### ISSUE THREAD

**Issue by erisvaldojunior** [mixpanel/mixpanel-react-native#117]
Seems like the library is not supporting nested objects as properties. Crash happens.

So for example:

```
// Works well

const gameLevel = {
      cardCount: 6,
      name: "Easy"
}

const gameDetails = {
     "Game Level": gameLevel
};

mixpanel.track(event, { ...gameDetails });

```

```

// Crashes because of grid object!

const gameLevel = {
      cardCount: 6,
      grid: { columns: 3, rows: 2 },
      name: "Easy"
}

const gameDetails = {
     "Game Level": gameLevel
};

mixpanel.track(event, { ...gameDetails });

```

**Comment by zihejia** [comment]
hi @erisvaldojunior , we will have a look.

**Comment by zihejia** [comment]
Hi @erisvaldojunior , this issue has been fixed in [1.3.5](https://github.com/mixpanel/mixpanel-react-native/releases/tag/v1.3.5). I'm closing this issue now, please feel free to reopen if the problem still exits.
